lib/inherit.js

```diff
@@ -68,6 +68,7 @@ function override(base, res, add) {
         }
         prop = add[name];
         if(isFunction(prop) &&
+                prop.prototype &&
```

Comment:
This `if` is used to wrap function to make `__base` work. Seems this won't work afterwards:
```js
{
  noPrototype() {
    this.__base();
  }
}
```